cypress/tests/ui-auth-providers/cognito.spec.ts

```diff
@@ -2,48 +2,84 @@ import "../../support/auth-provider-commands/cognito";
 import { isMobile } from "../../support/utils";
 
 if (Cypress.env("cognito_username")) {
-  describe("AWS Cognito", function () {
-    beforeEach(function () {
-      cy.task("db:seed");
+  // Sign in with AWS
+  if (Cypress.env("cognito_programmatic_login")) {
```

Comment:
any reason we can't do both?
```
;['ui', 'programmatic_login'].forEach((loginType) => {
	describe(`AWS Cognito - with ${loginType}`, function () {
             beforeEach(function () {
	        cy.task("db:seed");
		
		if (loginType === 'programmatic_login') {
                      cy.intercept("POST", "/bankAccounts").as("createBankAccount");
	         }
 	        
	        cy.loginByCognitoApi(Cypress.env("cognito_username"), Cypress.env("cognito_password"));
		cy.visit('/')
	      });

```

& Prob want to add cy.visit('/') give this will be broken once testIsolation is by default

Reply:
Honestly, due to how slow the realworld app was running in the first place with cognito, I was hesitant to put these too together. With that in mind, I tried to slim down some of the `cy.origin()` test examples and kept them separate for simplicity, even though I feel like the test config doesn't look simple at all 😅